internal/kafka/internal/services/clusters.go

```diff
@@ -736,7 +737,8 @@ func (c *clusterService) FindStreamingUnitCountByClusterAndInstanceType() (Kafka
 	var clusters []*ClusterSelection
 	dbConn := c.connectionFactory.New().
 		Model(&api.Cluster{}).
-		Where("status != ?", api.ClusterFailed)
+		Where("status != ?", api.ClusterFailed).
+		Where("cluster_type != ?", api.Enterprise.String())
```

Comment:
The implication of this exclusion is that:
1. enterprise data plane clusters won't be considering in dynamic scaling evaluation, this is fine.
2. But it also means that we won't have streaming unit metrics for enterprise data planes. I feel like this is something that might be needed. However, even the capacity used/max metrics for enterprise data plane might need to be revisited slightly a little. So I am happy with us capturing a JIRA about this so that we do not forget.

@pawelpaszki @pb82

Reply:
follow up Jira makes sense to me in this case

Reply:
I'll create one. Thanks